config_defaults/subtests/docker_cli/negativeusage.ini

```diff
@@ -66,8 +66,8 @@ stderr = Invalid hostPort
 
 [docker_cli/negativeusage/iv4]
 subcmd = run
-subarg = -e,PATH=/tmp,%%(FQIN)s,true
-stderr = exec\: \"true\"\: executable file not found in \$PATH
+subarg = -e,PATH=/tmp,%%(FQIN)s,tRuE
+stderr = .*tRuE.*not found
```

Comment:
No it wasn't.  I had a feeling there was something more to the breakage here.  Thanks for the ref. BZ.  I think a better "fix" would be for me to just stick that BZ in a comment and drop all the other changes.